packages/node_modules/@webex/internal-plugin-conversation/src/share-activity.js

```diff
@@ -219,7 +221,7 @@ const ShareActivity = WebexPlugin.extend({
           $uri(session) {
             return session.finishUploadUrl;
           },
-          body: {fileSize}
+          body: {fileSize, fileHash}
```

Comment:
Could you add a test to `unit/spec/share-activity.js` to validate the fileHash property is sent?

Reply:
Yes, I have added one test.